docs/releases/temporary-versions/version-v12.0.0b1.md

```diff
@@ -0,0 +1,205 @@
+
+By default, InvenioRDM allows metadata-only records. However, with the addition of the RDM_ALLOW_METADATA_ONLY_RECORDS configuration variable, you can now choose to disallow metadata-only records if desired, and any attempt to create or update a record without files or content will result in an error.
+
+To use this configuration variable, add it to your `invenio.cfg`:
+
+```python
+RDM_ALLOW_METADATA_ONLY_RECORDS = False
+```
+
+With this configuration option set to False, users cannot anymore create metadata-only records by omitting the files field when creating or updating a record.
+
+It's worth noting that allowing metadata-only records may have implications for data management and preservation, as well as for editing existing records (users will not be required to provide a file for each record. Therefore, it's recommended to carefully consider the use case and its potential consequences, before disabling this configuration option.
+
+#### Restricted records
+
+A restricted record is a record that contains sensitive or confidential information, and is not publicly accessible. Restricted records can be useful in cases where you want to provide access to certain information or resources to only a select group of users or organizations.
+
+By default, InvenioRDM allows restricted records. However, with the addition of the RDM_ALLOW_RESTRICTED_RECORDS configuration variable, you can now choose to disallow restricted records if desired, and any attempt to create or update a record with restricted access will result in an error.
+
+To use this configuration variable, add it to your `invenio.cfg`:
+
+```
+RDM_ALLOW_RESTRICTED_RECORDS = False
+```
+
+With this configuration option set to True, users cannot create records with restricted access by setting the access field to "restricted". The user interface will no longer show the option to restrict the record, and the rest API will return an error if the user has no sufficient permissions to perform this action. This will disable the restricted access and the users will be able to create only open access records.
+
+TODO: document the permissions here ?
+
+
+### New export formats
+
+We are pleased to announce that the latest release includes several new export formats to facilitate sharing and dissemination of research data. These new export formats offer a wider range of options for exporting metadata, making it easier to integrate with various third-party systems and to meet different requirements for data publication. The new export formats are available in addition to the existing export formats, providing more flexibility and options for researchers and data managers. We believe that these new additions will help advance the goal of open science and data sharing, and we hope that they will be useful to the research community.
+
+Detailed documentation of export formats is available [here](../../reference/export_formats.md) 
+
+### Data model extensions 
```

Comment:
In the past custom fields were called metadata extensions, putting data model extensions in the title might generate confusion...